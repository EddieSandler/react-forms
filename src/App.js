import './App.css';
import BoxList from "./BoxList"
import NewBoxForm from "./NewBoxForm"



function App() {
  return (
    <div className="App">
      <header className="App-header">


        <h1> Color Box Maker</h1>

        <BoxList />
  

      </header>
    </div>
  );
}

export default App;
