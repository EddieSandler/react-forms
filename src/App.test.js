import React from "react";
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';


import App from './App';

test('App renders', () => {
  render(<App />);
});
describe('App', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

