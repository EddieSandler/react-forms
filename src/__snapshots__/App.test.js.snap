// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1>
       Color Box Maker
    </h1>
    <div>
      <h3>
        Box List
      </h3>
      <div>
        <form
          onSubmit={[Function]}
        >
          <div>
            <label
              htmlFor="backroundColor"
            >
              Backround Color
            </label>
            <input
              id="backgroundColor"
              name="backgroundColor"
              onChange={[Function]}
              placeholder="background color"
              type="text"
              value=""
            />
          </div>
          <div>
            <label
              htmlFor="width"
            >
              width
            </label>
            <input
              id="width"
              name="width"
              onChange={[Function]}
              placeholder="width"
              type="text"
              value=""
            />
          </div>
          <label
            htmlFor="height"
          >
            Height
          </label>
          <input
            id="height"
            name="height"
            onChange={[Function]}
            placeholder="height"
            type="text"
            value=""
          />
          <div>
            <button>
              create box
            </button>
          </div>
        </form>
      </div>
      <div>
        <div
          className="Box"
          style={
            Object {
              "backgroundColor": "purple",
              "height": "250px",
              "position": "relative",
              "width": "250px",
            }
          }
        >
          <div>
            <button
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                  "position": "absolute",
                  "right": "0",
                  "top": "0",
                }
              }
            >
              x
            </button>
          </div>
        </div>
      </div>
    </div>
    <h1>
       Todo List
    </h1>
    <div>
      <h3>
        Todo List
      </h3>
      <div>
        <form
          onSubmit={[Function]}
        >
          <div>
            <label
              htmlFor="task"
            >
              Todo 
            </label>
            <input
              id="task"
              name="task"
              onChange={[Function]}
              placeholder="enter task"
              type="text"
              value=""
            />
          </div>
        </form>
      </div>
      <div />
    </div>
  </header>
</div>
`;
